docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/domain-model-layer-validations.md

```diff
@@ -97,11 +97,11 @@ Using field validation with data annotations, for example, you do not duplicate
   [*http://gorodinski.com/blog/2012/05/19/validation-in-domain-driven-design-ddd/*](http://gorodinski.com/blog/2012/05/19/validation-in-domain-driven-design-ddd/)
 
 - **Colin Jack. Domain Model Validation** \
-  [*http://colinjack.blogspot.com/2008/03/domain-model-validation.html*](http://colinjack.blogspot.com/2008/03/domain-model-validation.html)
+  [*https://colinjack.blogspot.com/2008/03/domain-model-validation.html*](https://colinjack.blogspot.com/2008/03/domain-model-validation.html)
```

Comment:
Here again, http instead of https is not being rendered:

![image](https://user-images.githubusercontent.com/10886961/50783460-ca7ea000-125f-11e9-9c39-b690cdbecd95.png)